akasha-vladan-main/css/style.scss

```diff
@@ -0,0 +1,76 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap');
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Open Sans', sans-serif;
+
+}
+
+.title-wrap {
+    background-color: black;
+    padding-top: 15px;
+    padding-bottom: 15px;
+}
+
+
+#title{
+    color: white;
+    text-align: center;
+}
+
+#img-div{
```

Comment:
Leave an indentation of one space before "{" 
so as to follow css best practices 😄

Reply:
- [ ] I think you should have phrased the comments politely, as suggestions instead of orders.

For example:

This is a polite suggestion: _"I think it would be better to use semantic tags"_

This reads as an order: _"Use semantic tags"_